public/pages/ChangePolicy/components/ChangeManagedIndices/ChangeManagedIndices.tsx

```diff
@@ -27,6 +27,7 @@ interface ChangeManagedIndicesProps {
   onChangeManagedIndices: (selectedManagedIndices: { label: string; value?: ManagedIndexItem }[]) => void;
   onChangeStateFilters: (stateFilter: { label: string }[]) => void;
   managedIndicesError: string;
+  core: CoreStart;
```

Comment:
Since I see you are wrapping all of the UI components in a core services consumer, you should be able to access the services via context (like how AD does it [here](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/master/public/pages/Dashboard/Container/DashboardOverview.tsx#L56)), rather than passing a `core` object via `props`. This would simplify your components from having to add a `core` prop for all components and/or subcomponents that need to access the core services for toasts, breadcrumbs, etc. More info [here](https://github.com/elastic/kibana/blob/7.9/src/core/MIGRATION.md#how-can-i-avoid-passing-core-services-deeply-within-my-ui-component-tree)

Reply:
In my mind, accessing object via Context or props depends on the usage frequency of the object.
If an object are utilized in most of the components from the parent to the child, passing that through `props` maybe looks better. While, if the object is only used in few components, especially the deepest component, accessing the object via `Context` like AD plugin does (and AD features in Alerting Kibana plugin as well) definitely much simpler.

Reply:
Changed to use `core = React.useContext(CoreServicesContext) as CoreStart;` to avoid passing core as props all the way from top level to lower level components.

Reply:
Modified code to get core from context instead of passing props.